cassandra/cassandra-docker.sh

```diff
@@ -175,114 +244,75 @@ else
 fi
 
 #Update the cassandra-env.sh with these new values
-cp /opt/apache-cassandra/conf/cassandra-env.sh.template /opt/apache-cassandra/conf/cassandra-env.sh
```

Comment:
Sorry, I can't help but do small "useless" changes when I'm still understanding the code. I hope this specific change can be seen as acceptable. If not, I'd have no problem in reverting this change.